src/IdentityServer/ResponseHandling/Default/DiscoveryResponseGenerator.cs

```diff
@@ -360,8 +360,8 @@ where scope.ShowInDiscoveryDocument
                 entries.Add(OidcConstants.Discovery.RequestUriParameterSupported, true);
             }
         }
-            
-        entries.Add(OidcConstants.Discovery.AuthorizationResponseIssParameterSupported, true);
+
+        entries.Add(OidcConstants.Discovery.AuthorizationResponseIssParameterSupported, Options.EmitIssuerIdentificationResponseParameter);
```

Comment:
I thought this was cherry picked from main back to the release branch? I guess we want it in main regardless...